Trees/Top view.cpp

```diff
@@ -0,0 +1,83 @@
+// C++ program to print top 
+// view of binary tree
+#include<bits/stdc++.h>
```

Comment:
please don't use **bits/stdc++.h** because this is quite Linux specific and slows down the compiler process. _(Specified in [README](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/README.md))_